<launch>
    <node name="scanAverager" pkg="rlracer" type="scanAverager.py" output="screen"/>
    <node name="GazeboListener" pkg="rlracer" type="GazeboListener.py" output="screen"/>
    <node name="RewardServer_noRestart" pkg="rlracer" type="RewardServer_noRestart.py" output="screen"/>
    <node name="CollisionDetector" pkg="rlracer" type="CollisionDetector.py" output="screen"/>
    <node name="StoreHumanData" pkg="rlracer" type="StoreHumanData.py" output="screen"/>
</launch>
